src/assets/colors.tsx

```diff
@@ -0,0 +1,14 @@
+export default {
+  gray: "#f8f8f8",
+  grayRegular: "#e5e5e5",
+  grayMedium: "#cccccc",
+  graySemibold: "#666666",
+  grayBold: "#333333",
+  purpleLight: "#e2d2ff",
+  purpleRegular: "#6e1fff",
+  redRegular: "#f7625a",
+  yellowRegular: "#fdefd1",
+  yellowMedium: "#fce1a8",
+  yellowBold: "#f1af1b",
+  greenRegular: "#0eda83"
+}
```

Comment:
I like this, but on the other hand it’s also nice to be able to just refer to the data in elan rather than having to keep it in sync. Do you have any thoughts on that?